cluster/gce/gci/configure-helper.sh

```diff
@@ -3046,6 +3046,11 @@ spec:
     - name: vol
       mountPath: /scrub
 EOF
+
+# fixup the alternate registry if specified
```

Comment:
note that `\EOF` with the `cat` command above prevents variable expansion, so we need a separate `sed` command after creating the file.